libs/hdf-converters/src/converters-from-hdf/asff/transformers.ts

```diff
@@ -487,7 +509,22 @@ function createProfileInfoFindingFields(
       }
     });
   });
+  const passThroughObj = _.get(hdf, 'passthrough');
+  if (_.isObject(passThroughObj)) {
+    typesArr = typesArr.concat(
+      pushSplitString(
+        escapeForwardSlashes(JSON.stringify(passThroughObj)),
+        ATTRIBUTE_CHARACTER_LIMIT,
+        'Execution/passthrough'
+      )
+    );
+  }
+  const typesArrLen = typesArr.length;
   typesArr = typesArr.slice(0, 50);
+  if (typesArrLen > 50) {
+    typesArr.pop();
+    typesArr.push('Execution/warning/Data has been lost via overflow');
```

Comment:
we should confirm with @ejaronne what specific language he'd like to put for this warning